localization/yabloc/yabloc_image_processing/launch/image_processing.launch.xml

```diff
@@ -1,7 +1,7 @@
 <launch>
   <arg name="src_image" default="/sensing/camera/traffic_light/image_raw"/>
   <arg name="src_info" default="/sensing/camera/traffic_light/camera_info"/>
-  <arg name="resized_image" default="/sensing/camera/undistorted/image_raw/relay"/>
+  <arg name="resized_image" default="/sensing/camera/undistorted/image_raw"/>
```

Comment:
Is this change related to this pull request aim?

Reply:
Sorry. This line is a minor change not directly related to the main objective. But it does not change the system behavior.
 I added this modification to the description of PR.

![image](https://github.com/autowarefoundation/autoware.universe/assets/24854875/6b195d03-1868-41e3-9105-2972f4c96cba)